coinstacks/bitcoin/api/nodemon.json

```diff
@@ -0,0 +1,6 @@
+{
+  "ignore": ["src/routes.ts"],
+  "watch": ["../../../packages/*/dist/tsconfig.tsbuildinfo", "../../common/api/dist/tsconfig.tsbuildinfo", "src"],
```

Comment:
This feels weird to me but maybe I don't understand what this is used for.

Reply:
This just watches for changes to common dependencies found in `packages` and `coinstacks/common`. I chose to use the `tsbuildinfo` file as this updates based on any changes to that packages `dist` output and reduces the number of file watches.